src/Symfony/Component/Console/Style/OutputStyle.php

```diff
@@ -51,15 +51,27 @@ public function createProgressBar(int $max = 0)
      */
     public function write($messages, bool $newline = false, int $type = self::OUTPUT_NORMAL)
     {
-        $this->output->write($messages, $newline, $type);
```

Comment:
outputs already support string|iterable at the contract level

Reply:
You are right, thanks